docs/CHANGELOG.rst

```diff
@@ -16,6 +16,7 @@ Added
 
 Changed
 -------
+- Use partial testing when running unit tests
```

Comment:
Improve the commit message to clarify when partial testing is used.

Reply:
Done.